quiz.py

```diff
@@ -1,6 +1,65 @@
 """
 File: quiz.py
-Author: Name
-Date: YYYY-MM-DD
-Description: A brief explanation of what this program does.
-"""
\ No newline at end of file
+Author: Sebastian Sanchez
+Date: 2024/03/04
+Description: Multiple choice quiz with final score
+"""
+
+print("-------------------------------")
+print("Multiple-Choice Quiz Game")
+
+#Initialize score
+score = 0
+
+#First question
+print("\n1. What is the power house of the cell?")
+print("(a) Nucleus")
+print("(b) Mitochondria")
+print("(c) Cell Membrane")
+
+first_answer = input("\n> ")
+
+#Is the first users answer right
+if first_answer == "b":
+    print("Correct!")
+    score = score + 1
+else:
+    print("Incorrect.")
+
+#Second question
+print("\n2. Who discovered gravity")
+print("(a) Isaac Newton")
+print("(b) Albert Einstein")
+print("(c) Charles Darwin")
+
+second_answer = input("\n> ")
+
+#Is the second users answer right
+if second_answer == "a":
+    print("Correct!")
+    score = score + 1
+else:
+    print("Incorrect.")
+
+#Third question
+print("\n3. Which element is a non-metal")
+print("(a) Iron")
+print("(b) Sodium")
+print("(c) Oxygen")
+
+third_answer = input("\n> ")
```

Comment:
You've used three different variables to get user input. Have you considered re-using the same variable again and again, just re-assigning it every time with a new `input()` statement? This makes for more memory-efficient code (better practice) since we are not doing anything with the individual choices at the end of the program. Imagine having to scale this program to 20 questions. That's a lot of extra variables to be stored in memory, without being used outside of the immediate comparison. In other words, as soon as you get to the next question, you don't really care anymore what the user input was for the previous one.

As a thought exercise, can you think of a program feature you could add where tracking individual choices/responses would be useful?